languages/elixir/exercises/concept/pattern-matching/mix.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Rules.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :pattern_match,
+      version: "0.1.0",
+      elixir: "~> 1.10",
```

Comment:
```suggestion
      # elixir: "~> 1.10",
```

We have been commenting this out to not penalize people who use the CLI with old versions, but the tests and test-runner will probably be built-maintained to latest minor release